server/controllers/finance/reports/index.js

```diff
@@ -13,7 +13,7 @@ exports.vouchers = require('./vouchers');
 exports.cash = require('./cash');
 exports.cashflow = require('./cashflow');
 exports.patient = require('./financial.patient').report;
-exports.incomeExpense = require('./incomeExpense');
+exports.income_expense = require('./income_expense');
```

Comment:
We don't have a naming convention for file name `./income_expense`, but for method i think that it was camelCase convention.